Starter.Core/Game.cs

```diff
@@ -14,7 +14,15 @@ public class Game
         /// <summary>
         /// How much time your snake has to respond to requests for this Game.
         /// Time is in milliseconds.
+        /// Example: 500
         /// </summary>
         public int Timeout { get; set; }
+
+
+        /// <summary>
+        /// Information about the ruleset being used to run this game.
+        /// Example: Example: {"name": "standard", "version": "v1.2.3"}
+        /// </summary>
+        public string Ruleset { get; set; }
```

Comment:
It would be better if Ruleset is represented by a class, not a string

Reply:
Agreed. I have updated the code and I believe it is now part of the PR.